api/WebGLRenderingContext.json

```diff
@@ -5693,7 +5693,7 @@
               "version_added": false
             },
             "samsunginternet_android": {
-              "version_added": false
+              "version_added": "12.0"
```

Comment:
This is another XR API, is it 12.0 or 11.2?

Reply:
Nice spot, thank you!!